src/main/java/com/google/sps/GetConfigProperties.java

```diff
@@ -6,6 +6,21 @@
 import java.util.Properties;
 
 public class GetConfigProperties {
+  public static String getPropertyValue(String filepath, String propertyName) throws FileNotFoundException, IOException {
+    String rootPath = Thread.currentThread().getContextClassLoader().getResource("").getPath();
+    String appConfigPath = rootPath + filepath;
+
+    Properties appProps = new Properties();
+    appProps.load(new FileInputStream(appConfigPath));
+
+    String propertyValue = appProps.getProperty(propertyName);
+    if (propertyValue == null) {
+      return "";
+    }
+    else
+      return propertyValue;
+  }
+
   public static String getApiKey() throws FileNotFoundException, IOException {
```

Comment:
Sorry I should have been a bit clearer, I did mean to have the getApiKey function call getPropertyValue()
that way in testing getPropertyValue you have also tested getApiKey and you don't have duplicated code.
So something like:
getApiKey() {
return getPropertyValue("app.properties", "api_key");
}